index.ts

```diff
@@ -149,20 +148,14 @@ const fetchOmnivore = async (
         }
 
         if (highlights?.length) {
-          const highlightBlock = await logseq.Editor.insertBlock(
-            articleBlock.uuid,
-            highlightTitle
-          )
-          if (!highlightBlock) {
-            throw new Error('block error')
-          }
-
-          for (const highlight of highlights) {
-            await logseq.Editor.insertBlock(
-              highlightBlock.uuid,
-              highlight.quote
-            )
-          }
+          const highlightBatch = highlights.map(it => {
+            return {
+              content: `${it.quote} - [read](https://omnivore.app/me/${slug}#${it.id})`
```

Comment:
Maybe we could mark the quote as a link like the article title
`[${it.quote}](https://omnivore.app/me/${slug}#${it.id})`

Reply:
Yeah I did try that but thought it was a little harder to read with the link underline.

Reply:
I think there is probably some ideal icon that would be more subtle.